Makefile

```diff
@@ -52,7 +52,7 @@ LDFLAGS += -extldflags "-static"
 endif
 
 .PHONY: all
-all: build test
+all: build test check
```

Comment:
also we should have these run in order from fast to slow: `build check testrace`